lib/setup.py

```diff
@@ -23,8 +23,6 @@
 
 VERSION = "1.28.1"  # PEP-440
 
-NAME = "streamlit"
```

Comment:
This will break nightly. See https://github.com/streamlit/streamlit/blob/develop/scripts/update_name.py#L37

Reply:
Updated the `update_name` script to work with the new name placing 👍